.github/workflows/regression.yml

```diff
@@ -29,6 +30,10 @@ jobs:
       - name: Install build-dependencies
         run: apt-get install -y liblz4-dev libreadline-dev zlib1g-dev libzstd-dev
 
+      - name: Install additional build-dependencies for 18+
```

Comment:
That is already committed in the other patch. Can you `rebase` to master please?

Reply:
Oh, right, my fault. I somehow was under the impression this is two different PR.